processor/src/main/java/org/mapstruct/ap/internal/model/source/BeanMapping.java

```diff
@@ -61,12 +63,25 @@ public static BeanMapping fromPrism(BeanMappingPrism beanMapping, ExecutableElem
             beanMapping.qualifiedByName(),
             resultTypeIsDefined ? beanMapping.resultType() : null );
 
-        return new BeanMapping(cmp, nullValueMappingStrategy );
+        //TODO Do we want to add the reporting policy to the BeanMapping as well? To give more granular support?
```

Comment:
Exactly. However, this is not yet exposed to `org.mapstruct.BeanMapping`. It is only used for our internally generated `ForgedMethods` where we don't apply name based mapping and therefore we need to explicitly ignore the unmapped target properties.

Exposing it in the public API via `org.mapstruct.BeanMapping` can be an option, but it is a different issue. I wrote the in order to see if you guys want to have something like this and whether I need to create an issue